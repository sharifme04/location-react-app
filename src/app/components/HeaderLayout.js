import React from "react";
import PropTypes from "prop-types";

const HeaderLayout = ({ children }) => {
  return <div className="headerContainer">{children}</div>;
};

HeaderLayout.propTypes = {
  children: PropTypes.any,
};

export default HeaderLayout;
