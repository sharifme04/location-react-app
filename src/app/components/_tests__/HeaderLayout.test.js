import React from "react";
import renderer from "react-test-renderer";
import HeaderLayout from "../HeaderLayout";

test("snapshot Form renders with empty props", () => {
  const component = renderer.create(<HeaderLayout />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot Form renders with  props", () => {
  const component = renderer.create(
    <HeaderLayout children={<div>this is an element</div>} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
