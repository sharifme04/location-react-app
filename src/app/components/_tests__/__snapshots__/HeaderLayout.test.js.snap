// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Form renders with  props 1`] = `
<div
  className="headerContainer"
>
  <div>
    this is an element
  </div>
</div>
`;

exports[`snapshot Form renders with empty props 1`] = `
<div
  className="headerContainer"
/>
`;
